css/style.css

```
@@ -1366,7 +1370,241 @@ a {
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--black);
+  text-transform: uppercase;
+  margin: 0;
+  margin-bottom: 14px;
+  padding: 11px 18px;
+}
+
+.product-filter {
+  width: 220px;
+}
+
+.filter-price,
+.filter-producers,
+.filter-power-supply {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.filter-price {
+  padding-bottom: 25px;
+  border-bottom: 2px var(--basic-background) solid;
+}
+
+.range-filter {
+  width: 220px;
+  margin-top: 11px;
+}
+
+.range-controls {
+  position: relative;
+  height: 41px;
+  margin-bottom: 10px;
+  padding: 40px 20px;
+  border-radius: 5px;
+  background-color: var(--options-background);
+}
+
+.scale {
+  height: 2px;
+  background-color: var(--filter-grey);
+}
+
+.bar {
+  width: 70%;
+  height: 2px;
+  background-color: var(--filter-green);
+}
+
+.toggle {
+  position: absolute;
+  top: 30px;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  padding: 0;
+  border: 8px solid var(--white);
+  background-color: var(--filter-toggle-grey);
+  border-radius: 50%;
+  box-shadow: 0 1px 1px 0 var(--shadow-black);
+  cursor: pointer;
+}
+
+.toggle-min {
+  left: 18px;
+}
+
+.toggle-max {
+  left: 135px;
+}
+
+.price-controls {
+  display: flex;
+  justify-content: space-between;
+}
+
+.price-del {
+  transform: translateY(25%);
+}
+
+.min-price,
+.max-price {
+  box-sizing: content-box;
+  font-family: "PT Sans", "Arial", sans-serif;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 18px;
+  color: var(--black);
+  text-transform: uppercase;
+  width: 75px;
+  padding: 8px 10px;
+  text-align: center;
+  border: none;
+  border-radius: 5px;
+  background-color: var(--options-background);
+}
+
+/* Убрал гадские стрелочки у числового input */
+.min-price::-webkit-outer-spin-button,
+.max-price::-webkit-outer-spin-button,
+.min-price::-webkit-inner-spin-button,
+.max-price::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+.min-price[type="number"],
+.max-price[type="number"] {
+  -moz-appearance: textfield;
+}
+/* ========================================== */
+
+.filter-producers {
+  margin: 14px 0;
+  padding-bottom: 13px;
+  border-bottom: 2px var(--basic-background) solid;
+}
+
+.filter-legend {
+  font-family: "PT Sans", "Arial", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 30px;
+  color: var(--black);
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.filter-list {
+  margin-top: 17px;
+}
+
+.filter-label {
+  font-family: "PT Sans", "Arial", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 19px;
+  color: var(--black);
+  text-transform: uppercase;
+  position: relative;
+  display: block;
+  cursor: pointer;
+  user-select: none;
+}
+
+.filter-input-checkbox + .filter-label::before {
+  content: "";
+  position: absolute;
+  left: -45px;
+  top: 0;
+  width: 23px;
+  height: 23px;
+  border: 2px var(--alt-grey-light) solid;
+  border-radius: 3px;
+}
+
+.filter-input-checkbox + .filter-label::before:hover {
```

можно просто вот так:
```suggestion
.filter-label:hover::before {
```

---

Исправлено